bridge/client/app/_interceptors/http-mock-interceptor.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+import {
+  HttpEvent,
+  HttpHandler,
+  HttpInterceptor,
+  HttpRequest,
+  HttpResponse
+} from "@angular/common/http";
+import {Observable, of} from "rxjs";
+import {map} from "rxjs/operators";
+
+const evaluationLabelsMockData = {
+  "testid": "12345",
+  "buildnr": "build17",
+  "runby": "JohnDoe"
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HttpMockInterceptor implements HttpInterceptor {
```

Comment:
Should this really be in this PR?

Reply:
Of course not. I removed the MockInterceptor.